envs/environment.yaml

```diff
@@ -55,5 +54,6 @@ dependencies:
   - tabula-py
 
   - pip:
+    - git+https://github.com/pypsa/pypsa.git#egg=pypsa
```

Comment:
I’ll do a release this week, so let us wait for this rather than pulling from master